### PR TITLE
Update migration guide of CE 2023.6.1

### PR DESCRIPTION
**Pull request by varmar05**

### CODE REVIEW

**Comment by tomasMizera** [comment]
Up for review now by @alex-cit and @varmar05

**Review by alex-cit — APPROVED** [review]

**Comment by alex-cit** [comment]
Looks great, thank you!
I have just fixed the links, hope you don't mind :)

File: src/dev/mergince.md [hunk 1]
```diff
@@ -27,18 +27,18 @@ $ sudo chmod g+s ./projects/
...
 $ docker exec merginmaps-server flask user create <username> <password> --is-admin --email <email>
 ```
 ​
-### Configure
+### Configure environment
 There are several application settings which can be changed via <GitHubRepo desc="config variables" id="MerginMaps/server/blob/master/.prod.env" />. Some of them have defaults and some of them needs to be modified, particularly those with `fixme` placeholder (marked with asterisks below).
 ​
 #### Server basics
-Settings about your deployment. Variables marked with star ⭐️ need to be modified.
+Variables marked with star ⭐️ need to be modified.
```
Comment (varmar05) [inline src/dev/mergince.md:41]: I think we should clarify what this really means. One thing is required settings without local deployment would not work (maybe we should make sure there is everywhere some default). Another thing is that we should clearly state which values should be modified for production deployment (e.g. default value is not safe, not recommended)

File: src/dev/mergince.md [hunk 2]
```diff
@@ -47,58 +47,64 @@ Settings about your deployment. Variables marked with star ⭐️ need to be mod
...
-#### Security settings
-Security settings. Variables marked with star ⭐️ need to be modified.
+#### Security settings 🛡️
+Security settings are important for production use.
 
 | Variable name            | Type      | Default   | Description |
 |--------------------------|-----------|-----------|-------------|
-| `SECRET_KEY`⭐️            | string    |           | Secret key for authorisation, should be generated strong string. |
-| `SECURITY_PASSWORD_SALT`⭐️| string    |           | Password salt for hashing, should be generated strong string. |
+| `SECRET_KEY`            | string    |           | Secret key for authorisation, should be generated strong string. |
```
Comment (varmar05) [inline src/dev/mergince.md:55]: this does not have default so it must be set?

File: src/dev/mergince.md [hunk 3]
```diff
@@ -47,58 +47,64 @@ Settings about your deployment. Variables marked with star ⭐️ need to be mod
 | `COLLECT_STATISTICS`     | Boolean   | `true`    | Whether to send usage statistics for application improvements. |
 | `SERVICE_ID`             | string    |           | Deployment UUID. Auto-generated on first run. |
 ​
-
-#### Security settings
-Security settings. Variables marked with star ⭐️ need to be modified.
+#### Security settings 🛡️
+Security settings are important for production use.
 
 | Variable name            | Type      | Default   | Description |
```
Comment (varmar05) [inline src/dev/mergince.md:53]: maybe we should sort them out alphabetically

File: src/dev/mergince.md (hunk 1 above)
Comment (tomasMizera) [reply src/dev/mergince.md:41]: Agreed, let me do that

File: src/dev/mergince.md (hunk 3 above)
Comment (tomasMizera) [reply src/dev/mergince.md:53]: Do you mean environment variables or these sections?

Comment (varmar05) [reply src/dev/mergince.md:53]: env variables for better readability

**Comment by tomasMizera** [comment]
> Looks great, thank you! I have just fixed the links, hope you don't mind :)

Sure, thanks for doing that @alex-cit !

**Comment by PeterPetrik** [comment]
2023.6.1 released , so we can merge this?

**Pull request merged**